src/NMEA/ClimbInfo.hpp

```diff
@@ -46,6 +46,9 @@ struct OneClimbInfo
   /** Altitude gained while in the thermal [m]. May be negative. */
   double gain;
 
+  /** Altitude lost while in the thermal [m]. */
+  double loss;
```

Comment:
Who writes this new field?

Reply:
Yes, I see that now. Too bad you made your diff so hard to read by doing two unrelated things in one commit, one of which required changing the indent of the code where you inserted new code. This makes your code change unreadable.